lib/msal-node/src/client/ManagedIdentityApplication.ts

```diff
@@ -43,7 +43,7 @@ export class ManagedIdentityApplication {
     private config: ManagedIdentityNodeConfiguration;
 
     private logger: Logger;
-    private nodeStorage: NodeStorage;
+    static nodeStorage: NodeStorage | undefined;
```

Comment:
1. Use `private static` to ensure visibility is correct.
2. Can't you use "null" instead of "undefined". Feels like you're propagating this "undefined" state in many places where "null" is a better abstraction. In this case, nodeStorage is always there, it's just that it's not initiatlized at first.

https://stackoverflow.com/questions/5076944/what-is-the-difference-between-null-and-undefined-in-javascript